index.js

```diff
@@ -109,24 +109,24 @@ function formatSettings(settings) {
 				if (settings.length > 0) {
 					// for each variable in module
 					settings.forEach((setting) =>
-						// append setting and default value
-						s += `--${setting}: ${module[setting]};\n`);
+										 // append setting and default value
+										 s += `--${setting}: ${module[setting]};\n`);
 				}
 			} else {
 				if (settings.length === 0) {
 					// fallback if module does not contain any options
 					// append module name and stub block
 					s += options.markdown
-						? `    ${_module} { /* no variables */ }\n`
-						: `    ${_module} { no variables }\n`
+						 ? `    ${_module} { /* no variables */ }\n`
+						 : `    ${_module} { no variables }\n`
```

Comment:
why just one

Reply:
I believe to align with 4-tab lengths? not 100% sure on that one tbh